### PR TITLE
Governance docs per CE PR 1226

### PR DESCRIPTION
**Pull request by duglin**
See: https://github.com/cloudevents/spec/pull/1226

### CODE REVIEW

File: MAINTAINERS.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Maintainers
+
+Current active maintainers of this SDK:
+
+- [Grant Timmerman](https://github.com/grant)
+- [Denys Makogon](https://github.com/denismakogon)
+- [Curtis Mason](https://github.com/cumason123)
+- [Claudio Canales](https://github.com/Klaudioz)
+- [Yurii Serhiichuk](https://github.com/xSAVIKx)
```
Comment (xSAVIKx) [inline MAINTAINERS.md:9]: I'm all for adding everyone, but I guess only Claudio and I are maintaining it now

**Review by xSAVIKx — APPROVED** [review]
@duglin I'm OK merging as is, just added a comment FYI

File: MAINTAINERS.md (hunk 1 above)
Comment (duglin) [reply MAINTAINERS.md:9]: Do you want to formally remove them as maintainers? I think a vote is needed per: https://github.com/cloudevents/spec/blob/main/docs/SDK-GOVERNANCE.md

Comment (xSAVIKx) [reply MAINTAINERS.md:9]: As I said, I don't mind people being mentioned, etc. But we can e.g. start some kind of a count-down and wait maybe half a year and see if they have any activity in the repo. What do you think?

Comment (duglin) [reply MAINTAINERS.md:9]: sounds reasonable to me

Comment (xSAVIKx) [reply MAINTAINERS.md:9]: we'll see how it goes then with this issue https://github.com/cloudevents/sdk-python/issues/222 😄

**Pull request merged**